src/systemctl/systemctl-show.c

```diff
@@ -298,12 +298,24 @@ static void format_active_state(const char *active_state, const char **active_on
                 *active_on = *active_off = "";
 }
 
+static void format_enable_state(const char *enable_state, const char **enable_on, const char **enable_off) {
+        if (streq_ptr(enable_state, "disabled")) {
+                *enable_on = ansi_highlight_red();
```

Comment:
The ultimate bikeshedding! Red is usually associated with an error, but disabled is not an error - maybe yellow?

Reply:
Sorry, I was meant to create another pull request so removed this one. Still trying to figure out git.

Reply:
no problem at all

Reply:
I guess, if I want to create new pull request while the previous one is still in progress, I have to checkout new branch?

Reply:
yes, just use a different branch (and base it on main, rather than on the other topic branch